# chassis
uint8 Sys_stADMd_mp
# 制动踏板开度，百分比
float64 VehDa_rBrkPedl_mp
# 整车总质量kg
float64 VehDa_mWght_mp
# 本车车速（km/h）（需显示在界面）
float64 VehDa_vEgoSpd_mp
# 本车加速度m/s2
float64 VehDa_aEgoAcc_mp
# 变速箱档位
int16 VehDa_stTraCurGear_mp
# TCU变速箱档位
int16 selected_gear_number
# 变速箱传动比
float64 VehDa_rTraCurGear_mp
# 离合器开关
uint8 VehDa_stCluSwt_mp
# 发动机摩擦扭矩百分比
int16 VehDa_prcTrqEngNomFric_mp
# 发动机预估损失扭矩百分比
int16 VehDa_prcTrqEstimdLoss_mp
# 制动设备源地址
uint8 VehDa_stSrcBrk_mp
# 发动机实际扭矩百分比
int16 VehDa_prcActuTrq_mp
# 驾驶员需求扭矩百分比
int16 VehDa_prcDrvrDmdTrq_mp
# 发动机控制设备源地址
uint8 VehDa_stSrcEngCtrl_mp
# 左前轮制动压力
float64 VehDa_pFrontLeft_mp
# 发动机转速
float64 VehDa_nEngSpd_mp
# 挂车状态
uint8 VehDa_stTrlrCnctn_mp
# 变速箱换挡状态
uint8 VehDa_stTraSht_mp
# 传动系统结合状态
uint8 VehDa_stTraEgd_mp
# 变速箱选择的档位
int16 VehDa_stTraSelGear_mp
# 变速箱TSC1限扭状态
uint8 VehDa_stTraTrqLim_mp
# 真实油门开度百分比
float64 VehDa_rAccrPedl_mp
# 位置跟踪误差
float64 SpdPln_lTrgLngErr_mp
# 变速箱输出轴转速
float64 VehDa_nOutpShaft_mp
# 制动系统准备可以释放
uint8 VehDa_stBrkReady4Rls_mp
